src/NUnitEngine/nunit.engine.core/Internal/AssemblyHelper.cs

```diff
@@ -68,7 +68,7 @@ private static bool IsFileUri(string uri)
         /// <summary>
         /// Gets the assembly path from code base.
         /// </summary>
-        /// <remarks>Public for testing purposes</remarks>
+        /// <remarks><para>Public for testing purposes</para></remarks>
```

Comment:
💡 Unrelated to this change, but I highly recommend using the `TestAccessor` pattern for this. It eliminates the need for testing concerns to impact the accessibility of members.